css-canvas-medical/index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="header">
+        <div class="header-logo">
+          <img src="images/logo-blue.svg" alt="canvas" class="logo">
+        </div>
+        <div class="header-nav">
```

Comment:
There is a `<nav>` element that you can use.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav